### DISCUSSION TITLE
Would that be possible to make the bot retrieve infos from battlemetrics ?

### DISCUSSION THREAD

**Discussion by Akouniza** [a-sync/game-server-watcher#12] [Ideas / requests]
Hey!

Would that be possible to make the bot retrieve infos from battlemetrics and such to retrieve players count from non steam api games?

Akou

**Comment by a-sync** [comment]
hey. 
gsw uses gamedig under the hood and connects directly to the game servers.  
steam api is just an optional fallback method.

**Reply by dkoz** [reply]
He's probably referring to games under Epic Online Services (EOS) like Squad and Ark: Survival Ascended. There is support for ASA on node-gamedig using epic protocol, but each game requires specific keys to contact the API. BattleMetrics already offers support for those specific games.

**Reply by a-sync** [reply]
both of those are available in gamedig tho 🤔  
anyway i dont plan on scraping server data from other services apart from steam atm. any new game support should be implemented in gamedig.
